extensions/wikia/Search/css/WikiaSearch.scss

```diff
@@ -41,14 +45,40 @@ $form-negative-side-padding: $form-side-padding*-1;
 		background: $color-search-form-background;
 		border-top: 1px solid $color-search-grid-border;
 		border-bottom: 1px solid $color-search-grid-border;
-		overflow: hidden;
 		padding: 9px 10px;
 
 		.inter-wiki-search & {
 			margin-top: 10px;
 			padding-bottom: 29px;
 		}
 
+		.wds-dropdown {
+			background-color: $color-page;
```

Comment:
did you try to use `@include dropdown-theming($color-page, $color-text, $color-links, $color-page-border);`? Not sure if I provided appropriate colors as arguments, but it should handle theming of wds-dropdown, including the hover state on list items which is currently only underlining the link

Reply:
I missed theming section on FDS, thanks!